images/send-svg.svg

```diff
@@ -0,0 +1,51 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" width="512" height="512">
+	<style>
+		tspan { white-space:pre }
+		.shp0 { fill: #ffffff } 
+	</style>
+	<g id="Layer">
+		<g id="Layer">
+			<path id="Layer" fill-rule="evenodd" class="shp0" d="M512 150.14L512 361.86C512 367.49 507.43 372.06 501.8 372.06L171.31 372.06C165.68 372.06 161.11 367.49 161.11 361.86L161.11 342.62L103.01 342.62C97.38 342.62 92.81 338.06 92.81 332.43C92.81 326.79 97.38 322.23 103.01 322.23L161.11 322.23L161.11 189.77L63.9 189.77C58.27 189.77 53.7 185.21 53.7 179.58C53.7 173.94 58.27 169.38 63.9 169.38L161.11 169.38L161.11 150.14C161.11 144.51 165.68 139.94 171.31 139.94L501.8 139.94C507.43 139.94 512 144.51 512 150.14ZM197.14 160.34L336.57 275.32L476.64 160.34L197.14 160.34ZM491.6 174.45L343.03 296.42C341.15 297.96 338.85 298.73 336.56 298.73C334.25 298.73 331.95 297.95 330.07 296.4L181.51 173.89L181.51 351.66L491.6 351.66L491.6 174.45Z" />
+		</g>
+	</g>
+	<g id="Layer">
+		<g id="Layer">
+			<path id="Layer" class="shp0" d="M119.67 244.77L10.2 244.77C4.57 244.77 0 249.33 0 254.97C0 260.6 4.57 265.17 10.2 265.17L119.67 265.17C125.31 265.17 129.87 260.6 129.87 254.97C129.87 249.33 125.31 244.77 119.67 244.77Z" />
+		</g>
+	</g>
+	<g id="Layer">
+		<g id="Layer">
+			<path id="Layer" class="shp0" d="M70.36 323.23L64.9 323.23C59.27 323.23 54.71 327.79 54.71 333.43C54.71 339.06 59.27 343.63 64.9 343.63L70.36 343.63C75.99 343.63 80.55 339.06 80.55 333.43C80.55 327.79 75.99 323.23 70.36 323.23Z" />
+		</g>
+	</g>
+	<g id="Layer">
+	</g>
+	<g id="Layer">
+	</g>
+	<g id="Layer">
+	</g>
+	<g id="Layer">
+	</g>
+	<g id="Layer">
+	</g>
+	<g id="Layer">
+	</g>
+	<g id="Layer">
+	</g>
+	<g id="Layer">
+	</g>
+	<g id="Layer">
+	</g>
+	<g id="Layer">
+	</g>
+	<g id="Layer">
+	</g>
+	<g id="Layer">
+	</g>
+	<g id="Layer">
+	</g>
+	<g id="Layer">
+	</g>
+	<g id="Layer">
+	</g>
+</svg>
```

Comment:
Czy to na pewno jest nam dalej potrzebne?

Reply:
No jeśli chcemy, żeby to wyglądało tak jak w Fignie to chyba tak, przynajmniej ja nie odnajduję takiej ikony w Font Awesome... jeśli źle szukam to mnie popraw proszę

Reply:
Okey, więc nie było pytania, dzięki